app/controllers/directory_objects_controller.rb

```diff
@@ -70,11 +70,12 @@ def placard
       name       = person.first + ' ' + person.last
       email = person.email
       department = person.department.title
-      title      = nil
+      title      = person.person_title
```

Comment:
Name of the relative. 'title' was already used for departments on the database so I called it person_title. I thought it'd be best to just follow the column name